metaworld/policies/sawyer_pick_place_v1_policy.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+
+from metaworld.policies.sawyer_push_v1_policy import SawyerPushV1Policy
+
+
+class SawyerPickPlaceV1Policy(SawyerPushV1Policy):
```

Comment:
please make sure every policy class is self-contained, i.e. inherits from Policy.

if someone only wants to know what the solution to `pick-place-v1` looks like, this inheritance only makes their lives harder.